openlibrary/core/lists/model.py

```diff
@@ -97,12 +97,17 @@ def get_editions(self, limit=50, offset=0, _raw=False):
 
         When _raw=True, the edtion dicts are returned instead of edtion objects.
         """
-        # Anand - Dec 2014: Disabled it is wasn't working from long time.
+        edition_keys = set([
```

Comment:
Converting to a set will mess with the ordering; does that matter here? Otherwise, can we just assume the universe is mostly kind, and that there are no duplicates? :)